server/auth.go

```diff
@@ -30,6 +33,11 @@ func AuthorizedToken(auth oauth2.Authenticator, logger chronograf.Logger, next h
 			WithField("method", r.Method).
 			WithField("url", r.URL)
 
+		if validSignature(log, r.Header.Get("message"), r.Header.Get("signature")) {
```

Comment:
What's standard practice for naming custom headers? `message` and `signature` seem a bit off.